@navikt/core/react/src/form/combobox/SelectedOptions/selectedOptionsContext.tsx

```diff
@@ -101,6 +102,14 @@ const SelectedOptionsProvider = ({
     [customOptions, onToggleSelected, removeCustomOption],
   );
 
+  useEffect(() => {
+    const nextIsLimitReached =
+      (!!maxSelected?.limit && selectedOptions.length >= maxSelected.limit) ||
+      !isMultiSelect;
+    setIsLimitReached(nextIsLimitReached);
```

Comment:
Do we need a state for `isLimitReached`?

Reply:
Yeah, the state is used a few places it seems (in filteredOptions), it was an existing state, i just refactored the assignment to use a variable, so i could re-use it in the same scope.

Reply:
I think you misunderstand. I should have added the comment on line 54 instead, because that's the line I'm really talking about 😅 You added the `setState()`, but I don't think it's needed. You can just move `const nextIsLimitReached` out of the `useEffect` and rename it back to "isLimitReached".

Reply:
It does seem to break without the state (I guess it forces a necessary rerender on change to render the limit reached in-list-banner?)

Reply:
actually, nvm, I think i must've forgotten to refresh, it seems to work now